src/frontend/views/Telegram/Telegram.module.css

```diff
@@ -0,0 +1,28 @@
+.heading {
+  composes: heading from '../../components/CredentialFlow/CredentialFlow.module.css';
+  background-image: url('../Attester/telegram.svg');
+}
+
+.buttonsLine {
+  text-align: center;
+}
+
+.iframe {
+  overflow: hidden;
+  border: medium none;
+  height: 2rem;
+  width: 16rem;
+}
```

Comment:
Can we center the contents of the iframe somehow?

Reply:
Nope, the internet security prevents us from doing that